src/SFA.DAS.DigitalCertificates.Domain/Interfaces/ISharingEntityContext.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.DigitalCertificates.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.DigitalCertificates.Domain.Interfaces
+{
+    public interface ISharingEntityContext : IEntityContext<Sharing>
+    {
+        Task<int> SaveChangesAsync(CancellationToken cancellationToken = default);
+
+        public async Task<List<Sharing>> GetAllSharings(Guid userId, Guid certificateId)
+        {
+            return await Entities
+                .AsNoTracking()
+                .Include(s => s.SharingAccesses)
+                .Include(s => s.SharingEmails!)
```

Comment:
SharingEmails is declared nullable but here it is guaranteed not null which wouldn't be the case